pkg/rancher-desktop/backend/k3sHelper.ts

```diff
@@ -1112,10 +1112,9 @@ export default class K3sHelper extends events.EventEmitter {
      * Check the given settings against the last-applied settings to see if we
      * need to restart the backend.
      * @param key The identifier to use for the UI.
-     * @param path The path in the Settings['kubernetes'] object to compare.
+     * @param path The path in the Settings object to compare.
```

Comment:
Although this was there before, `path` isn't one of the parameters... might as well fix it while we're here.